tests/nodeos_read_terminate_at_block_test.py

```diff
@@ -275,7 +276,10 @@ def executeSnapshotBlocklogTest(cluster, testNodeId, resultMsgs, nodeArgs, termA
     replayNodeosArgs = {
         5 : "--read-mode irreversible",
         6 : "--read-mode head",
-        7 : "--read-mode speculative"
+        7 : "--read-mode speculative",
+        8 : "--read-mode irreversible --force-all-checks",
+        9 : "--read-mode head --force-all-checks",
+        10 : "--read-mode speculative --force-all-checks"
```

Comment:
This verifies it doesn't SEGFAULT or error out. But it does not verify that all checks were actually performed.

Reply:
This type of test will come in via #727